src/vcSettings.cpp

```diff
@@ -221,6 +221,21 @@ bool vcSettings_Load(vcSettings *pSettings, bool forceReset /*= false*/, vcSetti
     pSettings->visualization.minIntensity = data.Get("visualization.minIntensity").AsInt(0);
     pSettings->visualization.maxIntensity = data.Get("visualization.maxIntensity").AsInt(65535);
 
+    const int classificationsCount = 256;
+    for (int i = 0; i < classificationsCount; i++)
```

Comment:
`udLengthOf(pSettings->visualization.customClassifcationToggles)` will avoid the need to have a magical constant - it will also allow the array to change size without any issues.

Reply:
Righto